conversation/backend/src/main/java/org/entcore/conversation/util/FolderUtil.java

```diff
@@ -50,72 +39,63 @@ public class FolderUtil {
      * @return a JsonArray representing the root folders of the hierarchical tree structure.
      */
     static public JsonArray listToTree(final JsonArray list, final int depth) {
-        // Sort root folders (with depth=1) by ID
-        SortedSet<JsonObject> rootFolders =  new TreeSet<>(compareOnIdAsc);
-        // Sort children folders (with depth=2+) by ID.
-        SortedSet<JsonObject>[] folderLevelById = new SortedSet[depth];
-        // Also sort children folders (with depth=2+) by parent folder ID
-        SortedSet<JsonObject>[] folderLevelByParentId = new SortedSet[depth];
+        // Sort the list, case-insensitive, but capitals are prioritized
+        List<JsonObject> sortedList = list.stream()
+            .map(obj -> (JsonObject) obj)
```

Comment:
On peut aussi écrire 
```suggestion
            .map(JsonObject.class::cast)
```
=> cela utilise la méthode `cast` de la classe `JsonObject` à la place de la function lambda codée sur-mesure (ici, `obj -> (JsonObject) obj`)